### ISSUE TITLE
Is it possible to have more than one `access key` (identity) and `access secret` (credential)?

### ISSUE THREAD

**Issue by thesuperzapper** [gaul/s3proxy#710]
My currently understanding is that it's only possible to have one HMAC access key and secret for each instance of `s3proxy`.

That is, you can set something like this:

```env
S3PROXY_AUTHORIZATION="aws-v2-or-v4"
S3PROXY_IDENTITY="__MY_CUSTOM_ACCESS_KEY__"
S3PROXY_CREDENTIAL="__MY_CUSTOM_ACCESS_SECRET__"
```

But there is no way to have more than one `access_key` and `access_secret` with multiple levels of permission (e.g. read only vs read write), this makes `s3proxy` less useful when trying to create an S3-API for Azure Blob Store.

Others seem to be confused by this also: https://github.com/gaul/s3proxy/issues/472

**Comment by gaul** [comment]
Yes -- as many front-end key pairs as you like in one process, one properties
file each. Closing as a duplicate of #364, which asked for the same thing.

```properties
# key-a.conf
s3proxy.endpoint=http://127.0.0.1:8080
s3proxy.authorization=aws-v2-or-v4
s3proxy.identity=__ACCESS_KEY_A__
s3proxy.credential=__ACCESS_SECRET_A__
s3proxy.bucket-locator.1=bucket-a
jclouds.provider=azureblob
jclouds.identity=...
jclouds.credential=...
```

```
s3proxy --properties key-a.conf --properties key-b.conf
```

Each key reaches only the buckets its own file claims: ListBuckets is filtered
per key, and another key's bucket is refused rather than merely hidden. Each
file carries its own backend as well, so the storage account behind each key
can differ. Only the server-level `s3proxy.*` settings -- endpoint, TLS,
virtual host, service path, CORS -- have to match across files.

What this does not give you is the second half of your question: two keys with
different rights over the *same* bucket. A bucket belongs to exactly one
configuration -- naming it in two files is refused at startup -- and
`s3proxy.read-only-blobstore` applies to a whole configuration, so read-only
for one key and read-write for another over one bucket is not expressible.
Please open a separate issue if you want that; it is a real gap rather than a
documentation problem.

For the front-end versus backend credential distinction you linked from #472,
@timuralp's comment there is the short version: the key your client presents
need not be the key S3Proxy presents to the backend.
